prettyprinter-ansi-terminal/prettyprinter-ansi-terminal.cabal

```diff
@@ -75,3 +75,6 @@ benchmark large-output
     ghc-options:         -O2 -rtsopts -Wall
     default-language:    Haskell2010
     type:                exitcode-stdio-1.0
+
+    if !impl(ghc >= 7.6)
+        build-depends: ghc-prim
```

Comment:
Could you add a comment that explains the need for this dependency?

Reply:
It mirrors 
https://github.com/quchen/prettyprinter/blob/e4a7f287c2e5057f096ad2a45c60052b87c83d23/prettyprinter/prettyprinter.cabal#L229-L230
Would you like me to add a comment to both?

Reply:
That would be great! I'm pretty puzzled about why we'd require `ghc-prim`.

Reply:
Done.

Reply:
Ah, that makes sense. Thanks! :)